src/pipeline.cc

```diff
@@ -1194,6 +1199,7 @@ NAN_METHOD(pipeline) {
   baton->flatten = AttrTo<bool>(options, "flatten");
   baton->negate = AttrTo<bool>(options, "negate");
   baton->blurSigma = AttrTo<double>(options, "blurSigma");
+  baton->medianSize = AttrTo<double>(options, "medianSize");
```

Comment:
The size is integral so `uint32_t` would make a better type here.

Reply:
fixed! sorry - again - for the copy-paste.